README.md

```diff
@@ -47,7 +47,7 @@ This example fetches the value of the key `foo` and passes it to the
 `var_dump` function. You can use any of the composition provided by
 [promises](https://github.com/reactphp/promise).
 
-If the key `foo` does not exist, the promise will be rejected.
+If the key `foo` does not exist, the promise will be resolve with null as value.
```

Comment:
[…] the promise will resolve with `null` as value.

Reply:
i'd be even more explicit:

[…] the promise will **fulfill** with `null` as value.